### PR TITLE
Decoding morse message 

### PR DESCRIPTION
**Pull request by houdamzari**
- [x] Creating a method to decode a Morse code character.
- [x] Creating a method to decode the entire message in Morse code.
- [x] Creating a method to decode an entire word in Morse code.
- [x] All linters are passing.

### CODE REVIEW

File: challenge.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  phrase_array = phrase.split
+  phrase_array.each { |section| word += morse_to_letter(section) }
+  word
+end
+
+def morse_to_message(message)
+  sentence = ''
+  message_array = message.split('   ')
+  message_array.each { |section| sentence += " #{morse_to_word(section)}" }
+  sentence
```
Comment (TedLivist) [inline challenge.rb:26]: - [ ] Let's try to return neat messages. This sentence decoder returns a decoded sentence with a leading whitespace. Kindly fix that. You can use `sentence.strip`

  image

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @houdamzari and @leehaney254,

Good job so far!
✔️ Reusable methods
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: challenge.rb (hunk 1 above)
Comment (leehaney254) [reply challenge.rb]: Thank you very much for the suggestion @TedLivist 👍

**Review by gemmen29 — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**